sql/core/src/main/scala/org/apache/spark/sql/execution/aggregate/ObjectAggregationIterator.scala

```diff
@@ -252,6 +249,22 @@ class SortBasedAggregator(
       var hasNextAggBuffer: Boolean = initialAggBufferIterator.next()
       private var result: AggregationBufferEntry = _
       private var groupingKey: UnsafeRow = _
+      /**
+       * A flag to represent how to process row for current grouping key:
+       *  0: update input row to aggregation buffer
+       *  1: merge input aggregation buffer to sort based aggregation buffer
```

Comment:
`sort based aggregation buffer`? Isn't it initial agg buffer?

Reply:
I changed to `sortBasedAggregationBuffer` make it clear